packages/react-components/src/organisms/TeamsTabbedCard.tsx

```diff
@@ -0,0 +1,75 @@
+import { TeamResponse } from '@asap-hub/model';
+import { network } from '@asap-hub/routing';
+import { css } from '@emotion/react';
+import React, { ComponentProps } from 'react';
+import { Link } from '../atoms';
+import { teamIcon } from '../icons';
+import { TabbedCard } from '../molecules';
+import { perRem, rem, tabletScreen } from '../pixels';
+
+const listItemStyle = css({
+  display: 'grid',
+  gridTemplateColumns: 'min-content 1fr',
+  gridColumnGap: `${9 / perRem}em`,
+});
+
+const containerStyles = css({
+  listStyle: 'none',
+  margin: 0,
+  padding: `${rem(32)} 0`,
+  display: 'grid',
+  rowGap: rem(12),
+  [`@media (min-width: ${tabletScreen.min}px)`]: {
+    gridTemplateColumns: '1fr 1fr',
+    columnGap: rem(15),
+  },
+});
+
+type TeamsTabbedCardProps = Pick<
+  ComponentProps<typeof TabbedCard>,
+  'title' | 'description'
+> & {
+  teams: ReadonlyArray<Pick<TeamResponse, 'id' | 'displayName'>>;
+};
+
+const TeamsTabbedCard: React.FC<TeamsTabbedCardProps> = ({
+  title,
+  description,
+  teams,
+}) => (
+  <TabbedCard
+    title={title}
+    description={description}
+    activeTabIndex={1}
+    tabs={[
+      {
+        tabTitle: 'Active Teams (0)',
+        items: [],
+        truncateFrom: 8,
+        disabled: true,
+      },
+      {
+        tabTitle: `Past Teams (${teams.length})`,
```

Comment:
I feel like it's a bit weird to filter inactive teams at the top and in here show them all 😛 so this case could happen where the counts don't match. 

<img width=250 src="https://user-images.githubusercontent.com/16595804/196661656-243d34b0-a3c3-4f87-84a5-24e9f6e7594a.png" />

Link to the page https://2180.hub.asap.science/network/groups/27b21989-8d51-44c5-8a5e-432efa740140/about#group-teams-83224df9-b0cd-4dd5-a0c5-652b8358d8a5 

Just leaving a comment here to ask if this was indeed expected 😉

Reply:
Good spot! @oratiu ticket removes the count in the header from inactive groups. This count reflects active teams.